benchmark/benchmark-cutest.jl

```diff
@@ -0,0 +1,79 @@
+include("config.jl")
+
+function get_status(code::Symbol) 
+    if code == :first_order
+        return 1
+    elseif code == :acceptable
+        return 2
+    else
+        return 3
+    end
+end
+
+@everywhere using CUTEst
+
+if SOLVER == "master"
+    @everywhere solver = nlp -> madnlp(nlp,linear_solver=MadNLPMa57,max_wall_time=900.,tol=1e-6)
+    @everywhere using MadNLP, MadNLPHSL
+elseif SOLVER == "current"
+    @everywhere solver = nlp -> madnlp(nlp,linear_solver=MadNLPMa57,max_wall_time=900.,tol=1e-6)
+    @everywhere using MadNLP, MadNLPHSL
+elseif SOLVER == "ipopt"
+    @everywhere solver = nlp -> ipopt(nlp,linear_solver="ma57",max_cpu_time=900.,tol=1e-6)
+    @everywhere using NLPModelsIpopt
+elseif SOLVER == "knitro"
+    # TODO
```

Comment:
If you don't have a Knitro's license, I think that something we could discuss with Artelys